kubeshop/tracetest
# add official Tracetest Dagger module & pokeshop example

This PR...

## Changes

this commit adds an initial version of the Tracetest Dagger module which enables
Dagger users to easily inject Tracetest in their CI pipelines. An example of
the pokeshop quickstart app is also providedo along with its corresponding github
action workflows

## Checklist



## How to test this PR: 

1. Download and install `Dagger` from https://docs.dagger.io/install/.
2. To see the official Dagger Tracetest module functions, call `dagger functions` at the root of this repository. 
3. To run the Dagger Pokeshop test example, run `dagger call tracetest --test-file ./tests/test.yaml` inside the `examples/quick-start-pokeshop/` directory.
